docs/e2e/e2e-checks.md

```diff
@@ -54,6 +56,20 @@ Then, run the tests with your app name and base url:
 make e2e-test-native APP_NAME=app BASE_URL=http://localhost:3000
 ```
 
+#### Running tests natively with shards
+
+The following commands split test execution into 3 separate shards, with results consolidated into a merged report located in `./e2e/blob-report`. This setup emulates how the sharded tests run in CI.
+```
+# ensure app is running on port 3000
+
+make e2e-test APP_NAME=app BASE_URL=http://host.docker.internal:3000 TOTAL_SHARDS=3 CURRENT_SHARD=1 CI=true && \
+make e2e-test APP_NAME=app BASE_URL=http://host.docker.internal:3000 TOTAL_SHARDS=3 CURRENT_SHARD=2 CI=true && \
+make e2e-test APP_NAME=app BASE_URL=http://host.docker.internal:3000 TOTAL_SHARDS=3 CURRENT_SHARD=3 CI=true
```

Comment:
can we try this
```suggestion
for shard in 1 2 3; do
    make e2e-test APP_NAME=app BASE_URL=http://host.docker.internal:3000 TOTAL_SHARDS=3 CURRENT_SHARD=$shard CI=true &
done
wait
```

or this

```
make e2e-test APP_NAME=app BASE_URL=http://host.docker.internal:3000 TOTAL_SHARDS=3 CURRENT_SHARD=1 CI=true &
make e2e-test APP_NAME=app BASE_URL=http://host.docker.internal:3000 TOTAL_SHARDS=3 CURRENT_SHARD=2 CI=true &
make e2e-test APP_NAME=app BASE_URL=http://host.docker.internal:3000 TOTAL_SHARDS=3 CURRENT_SHARD=3 CI=true &
wait
```

I think it'd be interesting to see if they can actually run in parallel locally as separate processes

Reply:
@lorenyu both of these don't work - I think we'd have to have a dynamic container name:

I think you'd need something like this:
```
docker run --rm \
    --name $(CONTAINER_NAME) \
    -e APP_NAME=$(APP_NAME) \
    -e BASE_URL=$(BASE_URL) \
    -e CURRENT_SHARD=$(CURRENT_SHARD) \
    -e TOTAL_SHARDS=$(TOTAL_SHARDS) \
    -e CI=$(CI) \
    -v /Users/ryan/Desktop/platform-test-nextjs/e2e/playwright-report:/e2e/playwright-report \
    -v /Users/ryan/Desktop/platform-test-nextjs/e2e/blob-report:/e2e/blob-report \
    playwright-e2e



for shard in 1 2 3; do
    make e2e-test APP_NAME=app BASE_URL=http://host.docker.internal:3000 TOTAL_SHARDS=3 CURRENT_SHARD=$shard CI=true CONTAINER_NAME=playwright-e2e-container-$shard &
done
wait
```

Reply:
Ah ok probably not worth it nevermind